Gems/AtomLyIntegration/CommonFeatures/Assets/Editor/Scripts/LegacyContentConversion/LegacyConversionHelpers.py

```diff
@@ -101,30 +101,36 @@ class Common_Command_Line_Options(object):
     """
     Some common options/parsing
     """
-    def __init__(self, argv0, argv1):
-        arguments = argv1.split('-')
-
+    def __init__(self, args):
         self.projectName = ""
+        self.assetCatalogOverridePath = ""
         self.includeGems = False
         self.useP4 = False
         self.endsWithStr = ""
         self.isHelp = False        
-        self.helpString = "usage: {0} -project=<project name> -include_gems -ends_with=<filter> -use_p4\n\
+        self.helpString = "usage: -project=<project name> -include_gems -ends_with=<filter> -use_p4 -assetCatalogOverridePath=<path>\n\
             E.g.:\n\
             {1} -project=StarterGame -include_gems\n\
```

Comment:
you will want to convert {1} to {args[0]} in order for the help text to show the command argument, or remove it.